roles/cifmw_cephadm/tasks/ceph_upgrade.yml

```diff
@@ -18,12 +18,15 @@
   ansible.builtin.set_fact:
     ceph_health: "{{ ceph.stdout | from_json }}"
 
-- name: Fail if health is HEALTH_WARN || HEALTH_ERR
+- name: Log ceph health for debug purposes
```

Comment:
If we proceed even when Ceph is `HEALTH_WARN`, do we care about printing this msg? When the process is over (if it doesn't fail) we trigger logs gathering (L67). Basically I think it doesn't get us useful information showing that the cluster is `HEALTH_OK` vs `HEALTH_WARN`.

Reply:
+1, if we go ahead without failing for HEALTH_WARN, we may avoid printing 'ceph -s -f json' here as we have same info collected in the logs at the end.

Reply:
However, looking at the last log it was sufficient, looking at the warning msg, to figure out the reason of the failed upgrade.
+1 to keep this task and only failing if `HEALTH_ERR`!

Reply:
I think it gets us useful information.

We didn't know why the job was failing before. We just knew it was in health warn. Now we know it's because OSD `count 2 < osd_pool_default_size 3`. 

If the job succeeds it will we gather and print the info but if it fails we don't get that information.

> When the process is over (if it doesn't fail) we trigger logs gathering (L67)

"if it doesn't fail" is the problem that part of this patch will solve.

It helps more than it harms.